boards/arm/ruuvi_ruuvitag/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Ruuvi-Ruuvitagboard configuration
```

Comment:
Small typo in commit message body, s/baord/board/

Reply:
Thanks, I'll fix it shortly

Reply:
@jukkar typo has been corrected.